client/src/components/Tip/Tip.js

```diff
@@ -1,7 +1,18 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Button, Popover, PopoverBody } from 'reactstrap';
+import { Popover, PopoverBody } from 'reactstrap';
 import i18n from 'i18n';
+import Button from '../Button/Button';
+import classNames from 'classnames';
+
+/**
+ * List of values that can be passed in the `type` prop of the `<Tip />` component.
+ * @type {{TITLE: string, TEXT_FIELD: string}}
+ */
+export const TIP_TYPES = {
+  TITLE: 'title',
```

Comment:
I think "title" makes more sense than say label because the the tip is next to the form field title, rather than the `<label>` html tag, which to me seems less obvious since it requires more knowledge of the implementation

If we change the name here we'd want to be consistent all the way back to `FormField::setLabelTip()`, which doesn't seems less obvious to me what it does